tests/yz_crdt.erl

```diff
@@ -113,6 +127,362 @@ confirm() ->
 
     pass.
 
+test_repeat_sets(Pid, Cluster, Bucket, Index, Key) ->
```

Comment:
Some prose (comments/additional logging) in these tests would be really helpful. 430 lines of added code with almost no explanation of what's happening is tough to review and tough to maintain. A 1-2 sentence comment at the beginning of each test, plus some additional logging of what you're expecting (not just what the outcome of the search was) would be greatly appreciated and make the tests more likely to be updated/fixed in the future if someone runs into issues.